src/execution_plan/ops/shared/create_functions.c

```diff
@@ -313,22 +313,12 @@ void PendingCreationsFree
 		pending->created_edges = NULL;
 	}
 
-	// free all graph-committed attributes associated with nodes
 	if(pending->node_attributes) {
-		uint prop_count = array_len(pending->node_attributes);
-		for(uint i = 0; i < prop_count; i ++) {
-			AttributeSet_Free(pending->node_attributes + i);
-		}
 		array_free(pending->node_attributes);
```

Comment:
Will `array_free` free each pointer allocated in the `AttributeSet *node_attributes`? Or were those allocated elsewhere, and this code (within this file/data structure) doesn't own these pointers?

Reply:
free only the array not each item as you can see free each item was deleted